addons/mass_mailing/models/mailing.py

```diff
@@ -518,12 +518,17 @@ def _action_view_traces_filtered(self, view_filter):
 
     def action_view_clicked(self):
         model_name = self.env['ir.model']._get('link.tracker').display_name
+        helper = (
+            _("No %s clicked your mailing yet!", model_name),
+            _("Link Trackers will measure how many times each link is clicked as well as the proportion of TargetModel who clicked at least once in your mailing.")
```

Comment:
Linting: maybe multi-line this sentence (going outside of github review = too long line ^^)